File: src/main/java/info/szadkowski/contact/properties/MailAddressesProperties.java
package info.szadkowski.contact.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties(prefix = "mail")
@Data
public class MailAddressesProperties {
  private String recipientMail;
  private String senderMail;
}

File: src/main/kotlin/info/szadkowski/contact/properties/MailAddressesProperties.kt
package info.szadkowski.contact.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

@ConstructorBinding
@ConfigurationProperties(prefix = "mail")
data class MailAddressesProperties(
    val recipientMail: String,
    val senderMail: String
)
